index.js

```diff
@@ -130,6 +158,9 @@ function Metric(namespace, units, defaultDimensions, options) {
   self._storedMetrics = [];
 
   if (self.options.enabled) {
+        if(!Object.keys(UNITS).find(u=>UNITS[u]===units)) {
```

Comment:
This line is indented too far. Also, if we're just asserting that the provided unit exists, and as we have already required `underscore`, what do you think of extracting the values from the object so than we could simply do a `_.contains` to check for the existence of the value in the object? Explicitly, we could extract the values via `const UNIT_VALS = _.values(UNITS)` and then our `if` condition simply becomes `if (_.contains(UNIT_VALS, units))`.